### PR TITLE
SENTRY - Return Sentry to page template

### PR DESCRIPTION
**Pull request by KamiQuasi**
Sentry, our script error tracking, wasn't on the site any more, so I put it back.
Environments now require the following to be set in their rhd.settings.yml file:
- sentry_track: true/false
- sentry_script: 'https://cdn.ravenjs.com/3.8.0/raven.min.js'
- sentry_code: 'https://cc00364690f241ffb2fcb39254d7f23f@sentry.io/115436' OR 'https://c3d1750237a549fe91ac70c27793181b@sentry.io/69051' for production

This will conditionally allow us to turn on or off the tracking, pinpoint a specific version of the script, and divide the tracking between environments.

### CODE REVIEW

**Comment by KamiQuasi** [comment]
@LightGuard or @Dantheman720 or @adelasofia big thing is to make sure there are no scripting errors, but also that in your local or the PR server that the "Raven" object is available in the Dev tools. We'll need to add the appropriate settings on Prod when this goes through.

**Comment by KamiQuasi** [comment]
@LightGuard or @Dantheman720 or @adelasofia give this one a look please so we can get script error reporting out. If you want to verify it just make sure the Raven object is available globally in the developer console and that there aren't any script errors.

File: _docker/environments/drupal-pull-request/rhd.settings.yml.erb [hunk 1]
```diff
@@ -19,4 +19,7 @@ database:
   username: 'drupal'
   password: 'drupal'
   name: 'drupal'
-dtm_code: //dpal-itmarketing.itos.redhat.com/developers
\ No newline at end of file
+dtm_code: //dpal-itmarketing.itos.redhat.com/developers
```
Comment (LightGuard) [inline _docker/environments/drupal-pull-request/rhd.settings.yml.erb]: Do we want this to be protocol agnostic or hard code it to https?

**Comment by Dantheman720** [comment]
@KamiQuasi - I checked the export site on this. I don't see the Raven object returning its value when I execute "Raven" in the console.

**Comment by KamiQuasi** [comment]
@Dantheman720 just to make sure the way I'm seeing the Raven object is go to the exported site, bring up the developer console and type in "Raven" which should present some type of object to you. I tried it in Chrome and Firefox, and see it. Is that how you're doing it?

**Comment by adelasofia** [comment]
@KamiQuasi I can see the Raven object on the Exported site

**Comment by LightGuard** [comment]
@KamiQuasi @adelasofia @Dantheman720 Are we good to merge now?

**Comment by KamiQuasi** [comment]
@LightGuard I vote yes. Also, once merged we'll need the rhd.settings.yml file updated with the values mentioned above on production to actually make the script get put on pages.

**Comment by KamiQuasi** [comment]
@LightGuard when you have a chance today please go ahead and merge this and put the config values on the production environment please.

**Comment by LightGuard** [comment]
@KamiQuasi Sorry, there's a merge conflict now, please fix

**Comment by Dantheman720** [comment]
@KamiQuasi @LightGuard @adelasofia - I see the object as well. The merge conflicts have also been fixed.

**Pull request merged**